cmd/nodecmd/create_devnet.go

```diff
@@ -41,7 +43,7 @@ const (
 
 func generateCustomCchainGenesis() ([]byte, error) {
 	cChainGenesisMap := map[string]interface{}{}
-	cChainGenesisMap["config"] = coreth_params.AvalancheLocalChainConfig
+	cChainGenesisMap["config"] = coreth_params.GetChainConfig(avago_upgrade.GetConfig(avago_constants.LocalID), coreth_params.AvalancheLocalChainID)
 	cChainGenesisMap["nonce"] = hexa0Str
 	cChainGenesisMap["timestamp"] = hexa0Str
```

Comment:
did you tested this genesis works? this is not the same solution as found for ANR:
```
conf := *coreth_params.TestChainConfig
conf.ChainID = coreth_params.AvalancheLocalChainID
cChainGenesisMap["config"] = conf
cChainGenesisMap["timestamp"] = upgrade.InitiallyActiveTime.Unix()
```
but may work. You should check if teleporter msg works between c-chain and other blockchain.

Reply:
it's after update to latest coreth